src/GraphNodeManagerViewExtension/GraphNodeManagerViewModel.cs

```diff
@@ -170,9 +171,10 @@ public bool IsAnyFilterOn
         /// Establish the Current Workspace
         /// </summary>
         /// <param name="p"></param>
-        public GraphNodeManagerViewModel(ViewLoadedParams p, string id)
+        public GraphNodeManagerViewModel(ViewLoadedParams p, string id, Action<Logging.ILogMessage> logDelegate)
```

Comment:
is the viewModel ok to break public signature ?

Reply:
This is recently added. Not part of previous release.

Reply:
ok